src-ui/src/app/components/common/filterable-dropdown/filterable-dropdown.component.ts

```diff
@@ -15,6 +15,8 @@ export class FilterableDropdownSelectionModel {
   changed = new Subject<FilterableDropdownSelectionModel>()
 
   multiple = false
+  private _logicalOperator = 'and'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592508)
```suggestion
  private _logicalOperator = "and"
```